package.json

```diff
@@ -26,6 +26,8 @@
     "clean": "gts clean",
     "compile": "tsc",
     "fix": "gts fix",
-    "test": "node --experimental-top-level-await --experimental-loader ./src/loader.mjs ../presm/examples/tests/tests.ts"
+    "test": "npm run test-unit",
+    "test-ts": "node --experimental-top-level-await --experimental-loader ./src/loader.mjs ../presm/tests/tests.ts",
```

Comment:
Not a blocker for this PR, could be a follow-up: I think it would be better to clearly split the test from the test subject ("the thing being tested"). E.g. you can create a test that runs without a loader and that spawns one (or more) node processes with the loader and tests the resulting behavior.

Reply:
I think I see what you mean but will follow up during our sync tomorrow